tests/network/migration/test_masquerade_connectivity_after_migration.py

```diff
@@ -57,7 +57,24 @@ def running_vm_for_migration(
 
 
 @pytest.fixture()
-def migrated_vmi(running_vm_for_migration):
+def inter_vm_connectivity_exists(
+    running_vm_static,
```

Comment:
(not in this PR but `running_vm_static` should be renamed to something meaningful)

Reply:
Ack, I'll change it in the refactoring PR (https://github.com/RedHatQE/openshift-virtualization-tests/pull/1908), and also change the other VM accordingly (running_vm_for_migration).

Reply:
https://github.com/RedHatQE/openshift-virtualization-tests/pull/1908/commits/8c099a536bc4e108ed508431ae9d246c6f4b1641